tokio/src/loom/std/atomic_u32.rs

```diff
@@ -2,7 +2,6 @@ use std::cell::UnsafeCell;
 use std::fmt;
 use std::ops::Deref;
 
-/// `AtomicU32` providing an additional `load_unsync` function.
```

Comment:
This still provides `unsync_load` - should the comment be updated instead of removed?

Reply:
Sorry, my mistake -- Will update.